mesonbuild/compilers/swift.py

```diff
@@ -27,12 +27,12 @@
                            }
 
 class SwiftCompiler(Compiler):
-    def __init__(self, exelist, version):
+    def __init__(self, exelist, version, for_machine: MachineChoice):
         self.language = 'swift'
-        super().__init__(exelist, version)
+        super().__init__(exelist, version, for_machine)
         self.version = version
         self.id = 'llvm'
-        self.is_cross = False
+        assert for_machine <= MachineChoice.HOST
```

Comment:
Lets not use rich comparisons with this enum. It's not extremely clear what is "greater" and what is "less", and given that there are 3 choices (and the chances of adding another choice are *extremely* low), this is always just `if x is not a` or `if x is c`

Reply:
As above, I'm just removing this. As there was no assert before, and hey, with these changes cross might just start working for the first time.